geode-core/src/main/java/org/apache/geode/distributed/internal/LocatorLoadSnapshot.java

```diff
@@ -448,14 +486,28 @@ private void updateMap(Map map, ServerLocation location, float load, float loadP
     }
   }
 
+  private void updateMap(Map map, ServerLocation location, String memberId, float load,
```

Comment:
This is a new method and should be, at leat, unit tested.
You can make it package private, annotate it with `@VisibleForTesting` and access it directly from `LocatorLoadSnapshotJUnitTest` and/or `LocatorLoadSnapshotIntegrationTest`.

Reply:
done